pkg/tpm/tpm.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2015 CoreOS, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build tpm
+
+package tpm
+
+import (
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/coreos/go-tspi/tpmclient"
+)
+
+// Extend extends the TPM log with the provided string. Returns any error.
+func Extend(desription string) error {
```

Comment:
Is there anything preventing us from adding the deps in semaphore and
building this there?

On Tue, Nov 24, 2015 at 1:43 PM, Krzesimir Nowak <notifications@github.com>
wrote:

> In pkg/tpm/tpm.go
> <https://github.com/coreos/rkt/pull/1775#discussion_r45799081>:
>
> > +//
> > +// Unless required by applicable law or agreed to in writing, software
> > +// distributed under the License is distributed on an "AS IS" BASIS,
> > +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +// See the License for the specific language governing permissions and
> > +// limitations under the License.
> > +// +build tpm
> > +
> > +package tpm
> > +
> > +import (
> > +	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/coreos/go-tspi/tpmclient"
> > +)
> > +
> > +// Extend extends the TPM log with the provided string. Returns any error.
> > +func Extend(desription string) error {
>
> Didn't notice it before - there is a missing *c* in parameter name -
> should be des*c*ription. We don't build it for now, so it wasn't caught
> by CI before.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/1775/files#r45799081>.
>